camp/abolotski/XBoxToTalon/simgui-ds.json

```diff
@@ -0,0 +1,97 @@
+{
```

Comment:
wondering if we need to checkin this file, or if it should be in .gitignore?

Reply:
added to .gitignore